pandas/tests/io/formats/test_to_html.py

```diff
@@ -415,6 +431,85 @@ def test_to_html_multiindex_max_cols(self, datapath):
         expected = expected_html(datapath, 'gh6131_expected_output')
         assert result == expected
 
+    def test_to_html_multi_indexes_index_false(self, datapath):
+        # GH 22579
+        df = DataFrame({'a': range(10), 'b': range(10, 20), 'c': range(10, 20),
+                        'd': range(10, 20)})
+        df.columns = MultiIndex.from_product([['a', 'b'], ['c', 'd']])
+        df.index = MultiIndex.from_product([['a', 'b'],
+                                            ['c', 'd', 'e', 'f', 'g']])
+        result = df.to_html(index=False)
+        expected = expected_html(datapath, 'gh22579_expected_output')
+        assert result == expected
+
+    @pytest.mark.parametrize(
+        ('idx_flag', 'named_index', 'expected_output'),
+        [(True, True, 'gh22579_truncation_index_true'),
+         (False, True, 'gh22579_truncation_index_false_named'),
+         (False, False, 'gh22579_truncation_index_false_unnamed')
+         ])
+    def test_to_html_multi_indexes_index_false_with_truncation(
+            self, datapath, idx_flag, named_index, expected_output):
+        # GH 22579 with truncation
+        names = ['foo', None, 'baz'] if named_index else None
+        index = MultiIndex.from_product([['a', 'b'], ['c', 'd'], ['e', 'f']],
+                                        names=names)
+        df = DataFrame(np.arange(64).reshape(8, 8), index=index, columns=index)
+        result = df.to_html(max_rows=4, max_cols=4, index=idx_flag)
+        expected = expected_html(datapath, expected_output)
+        assert result == expected
+
+    @pytest.mark.parametrize('col_idx_type',
+                             [Index([0, 1]),
```

Comment:
Style nit but can you move the opening bracket up one line here? Should allow for better indentenation of the arguments

Reply:
much better! thanks.